templates/pkg/resource/manager.go.tpl

```diff
@@ -129,11 +129,12 @@ func (rm *resourceManager) Update(
 }
 
 // Delete attempts to destroy the supplied AWSResource in the backend AWS
-// service API.
+// service API, returning an AWSResource representing the
+// resource being deleted (if delete is asynchronous and takes time)
 func (rm *resourceManager) Delete(
 	ctx context.Context,
 	res acktypes.AWSResource,
-) error {
+) (acktypes.AWSResource, error) {
```

Comment:
you cannot change this method signature before changing the interface in ACK runtime:

https://github.com/aws-controllers-k8s/runtime/blob/main/pkg/types/aws_resource_manager.go#L61-L63

Reply:
PR: https://github.com/aws-controllers-k8s/runtime/pull/21 contained changes in runtime.

Reply:
Updated the PR summary as: "Returning the latest resource from the Delete() call"